tests/devices/test_default_qubit_torch.py

```diff
@@ -192,6 +192,17 @@ def test_analytic_deprecation():
 class TestApply:
     """Test application of PennyLane operations."""
 
+    def test_conj_array(self, device, torch_device, tol):
```

Comment:
Test added for coverage?

Reply:
Yes exactly, as we remove the use of `conj` in the torch device.